packages/aws-cdk-lib/aws-appsync/lib/graphqlapi.ts

```diff
@@ -442,6 +442,20 @@ export interface GraphqlApiProps {
    * @default IntrospectionConfig.ENABLED
    */
   readonly introspectionConfig?: IntrospectionConfig;
+
+  /**
+   * A number indicating the maximum depth resolvers should be accepted when handling queries.
+   *
+   * @default - No Limit
+   */
+  readonly queryDepthLimit?: number;
+
+  /**
+   * A number indicating the maximum number of resolvers that should be accepted when handling queries.
+   *
+   * @default - No Limit
```

Comment:
https://docs.aws.amazon.com/appsync/latest/APIReference/API_GraphqlApi.html

"The default value is 0 (or unspecified), which will set the limit to 10000"

Reply:
Added